pyccel/parser/semantic.py

```diff
@@ -446,13 +446,15 @@ def get_import(self, name):
         """."""
 
         imp = None
-
         container = self.namespace
         while container:
 
-            if name in container.imports['imports']:
-                imp =  container.imports['imports'][name]
-                break
+            #case import x as y
```

Comment:
You know this is going to fall pray to "You touched it you document it right" ^^